File: CS/UnobtrusiveValidation/Controllers/HomeController.cs
using System;
using System.Web;
using System.Web.Mvc;
using JQueryValidation.Models;
using DevExpress.Web.Mvc;

namespace JQueryValidation.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return UnobtrusiveValidation();
        }
        [AcceptVerbs(HttpVerbs.Get)]
        public ActionResult UnobtrusiveValidation() {
            return View("UnobtrusiveValidation", new ValidationData());
        }
        [AcceptVerbs(HttpVerbs.Post)]
        public ActionResult UnobtrusiveValidation([ModelBinder(typeof(DevExpressEditorsBinder))] ValidationData validationData) {
            if(ModelState.IsValid)
                return View("UnobtrusiveValidationSuccess");
            return View("UnobtrusiveValidation", validationData);
        }
        public ActionResult UnobtrusiveValidationSuccess() {
            return View("UnobtrusiveValidationSuccess");
        }
    }
}

File: VB/UnobtrusiveValidation/Controllers/HomeController.vb
Imports System.Web.Mvc
Imports JQueryValidation.Models
Imports DevExpress.Web.Mvc

Namespace JQueryValidation.Controllers

    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return UnobtrusiveValidation()
        End Function

        <AcceptVerbs(HttpVerbs.Get)>
        Public Function UnobtrusiveValidation() As ActionResult
            Return View("UnobtrusiveValidation", New ValidationData())
        End Function

        <AcceptVerbs(HttpVerbs.Post)>
        Public Function UnobtrusiveValidation(<ModelBinder(GetType(DevExpressEditorsBinder))> ByVal validationData As ValidationData) As ActionResult
            If ModelState.IsValid Then Return View("UnobtrusiveValidationSuccess")
            Return View("UnobtrusiveValidation", validationData)
        End Function

        Public Function UnobtrusiveValidationSuccess() As ActionResult
            Return View("UnobtrusiveValidationSuccess")
        End Function
    End Class
End Namespace
